biorbd_optim/problem_type.py

```diff
@@ -5,7 +5,6 @@
 from .plot import CustomPlot
 from .enums import PlotType
 
-
```

Comment:
Please make sure you respect PEP recommendations (having two spaces between the imports and the class declarations)